docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+where we got the <Tooltip text={<strong>strings</strong>} tooltip= "A key that links to text in the LOCR File" /> from. 
+Both will be using the same <Tooltip text={<strong>'Syntax'</strong>} tooltip="A set of rules that define how code is written and organized." />.
+
+> This was visually represented as a *simplified LOCR file* at the [beginning of Part 2](#pt-2-get-your-string).
+
+
+*Even better too if you have noticed how similar this method is to [the previous *blobs article*](addingblobs.md#step-3-set-up-a-blobsfolder) where we 'recreated'* <Tooltip text={<strong>an assembly path</strong>} tooltip="Bonus points if you connected that HASH lines are 'coded' assembly paths = filePaths + fileNames." />
+:::
+</details>
+</div>
+
+### 5. (Optionally) Add more languages
+#### Same Strings:
+In [Part 2](#2-copy-the-string) of the tutorial we demonstrated that *strings* are the same in each <Tooltip text={<><strong>language array</strong><span className="icon-alert"></span></>} tooltip="A single language array in a LOCR file consisting of strings" />
+>*This will make adding languages for each entry very easy.*
+    
+
+To have your mod support more languages, simply:
+- make another <Tooltip text={<strong>language bracket</strong>} tooltip="A single language array that can contain multiple entries of strings each attached to text you see in-game" /> 
+- add the <Tooltip text={<strong>same decimal</strong>} tooltip="A string converted into a numeral form" /> with your text
+
+
+Example of a new language line:
+<div id="langcode"></div>
+```json
+"localisationOverrides": {
+        "yourLocrHASH": {
+            "english": {
+                "yourconverteddecimal": "New Suit Name"
+            },
+            "french": {
+                "yourconverteddecimal": "Niveua Costume nome"
+            }
+        }
+    }
+```
+<details>
+<summary>Language Codes</summary>
+
+:::info language codes
+- The reason why `french` is used here and not `français` is because the manifest's <Tooltip text={<strong>schema</strong>} tooltip="A structure used to validate and provide auto-completion for your manifest, helping you write error-free code." /> is coded in English. 
+- As for why it doesn't use the same <Tooltip text={<strong>language codes</strong>} tooltip="The language identifiers in each language array of a LOCR file" /> as the actual <Tooltip text={<strong>LOCR Files</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> like `en` or `fr`, is because the manifest's <Tooltip text={<strong>schema</strong>} tooltip="A structure used to validate and provide auto-completion for your manifest, helping you write error-free code." /> is coded like that. 
```

Comment:
"As for why it doesn't use the same as the other file, that's because that's the way it is" is not exactly super helpful. I don't really have a better explanation but this whole block doesn't really add anything

Reply:
Well I just had to find a way to emphasize it. I can remove the reason for why it its the way it is after